server/mailer.js

```diff
@@ -108,14 +112,22 @@ export class Mailer {
     });
   };
 
-  export = async (opts: { to: string, attachments: Object[] }) => {
+  export = async (opts: {
+    to: string,
+    attachments?: Object[],
+    id: string,
+    state: string,
+  }) => {
     this.sendMail({
       to: opts.to,
       attachments: opts.attachments,
       title: "Your requested export",
       previewText: "Here's your request data export from Outline",
```

Comment:
These don't make much sense if it failed, we might need to split the mailer into two to be honest.